core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -738,8 +740,10 @@ private void handleLocaleCheck() {
     unbinder.unbind();
     // TODO create a base Activity class that class this.
     FileUtils.deleteCachedFiles(getActivity());
+    if (tts != null){
     tts.shutdown();
```

Comment:
this is not indented properly

Reply:
this is still not indented properly, the body of an if shouldn't be at the same level as the if statement